src/test/scala/com/crealytics/spark/v2/excel/WriteAndReadSuite.scala

```diff
@@ -72,6 +72,12 @@ object WriteAndReadSuite {
     Row(4, "2021-11-11", "2021-11-11 01:23:05"),
     Row(5, "2022-10-01", "2022-10-01 16:23:45")
   )
+
+  val expectedData_bool: List[Row] = List(Row(1, true), Row(2, false))
```

Comment:
Could you check if the tests correcly uncover the missing functionality if you add a bool to `userDefinedSchema_01`?
That would save a lot of code (the entire `write then read boolean` test).

Reply:
@nightscape updated the test code